pkgs/top-level/all-packages.nix

```diff
@@ -10553,6 +10553,8 @@ in
 
   gdal_1_11 = callPackage ../development/libraries/gdal/gdal-1_11.nix { };
```

Comment:
I think that, in a separate PR, we should remove this version.

Reply:
I was flirting with the idea with removing it, but it's "used" by gplates (which is broken right now on master), tried updating the gplates package, but ran into some GL header issues and it didn't seem worth the effort.

Reply:
No need to bother with packages that are broken and unmaintained.

Reply:
Fair, I'm still at the point with nix where it's just interesting to poke around and see how it works. Learn a lot about linux internals, and different ways to do builds as well.